bittensor/__init__.py

```diff
@@ -208,7 +208,7 @@ def debug(on: bool = True):
 from .chain_data import *
 from .subtensor import subtensor as subtensor
 from .cli import cli as cli, COMMANDS as ALL_COMMANDS
-from .logging import logging as logging
+from .btlogging import logging as logging
```

Comment:
so this maintains the functionality for `bittensor.logging`, correct?

Reply:
Yep, only changes the import. Can still call `bt.logging.<>`.

It's basically covering the edge case in jupyter notebooks where you're running python from the same directory.